exec/cnex/object.c

```diff
@@ -108,8 +109,8 @@ void object_releaseObject(Object *o)
         o->refcount--;
 
         if (o->refcount <= 0) {
-            if (o->cell != NULL) {
-                cell_freeCell(o->cell);
+            if (o->ptr != NULL) {
+                cell_freeCell(o->ptr);
```

Comment:
This doesn't look right. What if `ptr` isn't a cell? Can that happen here?

Reply:
No, it shouldn't EVER happen here.  But it is actually the responsibility of the caller to make sure that the default object_releaseObject() never gets called on any non-cell objects.  I have no way to test to see if the object is a Cell or not either.  Unless I add custom enums for the Objects types.